public/app/features/scenes/dashboard/LibraryVizPanel.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import { SceneComponentProps, SceneObjectBase, SceneObjectState, VizPanel } from '@grafana/scenes';
+import { LibraryPanel, LibraryPanelRef } from '@grafana/schema';
+import { PanelModel } from 'app/features/dashboard/state';
+import { getLibraryPanel } from 'app/features/library-panels/state/api';
+
+import { createPanelDataProvider } from './utils/createPanelDataProvider';
+
+interface LibraryVizPanelState extends SceneObjectState {
+  title: string;
```

Comment:
Seems like Library Panels respect not the original lib anel title, but actual panel title from dashboard JSON. Passing this here as this seems to be the only required property other than library panel ref.

Reply:
I think this is a good note to add as comment 😃